### PR TITLE
[MERGE] Merge branch-1.6

### PR DESCRIPTION
**Pull request by aaudiber**
Versions fixed in second to last commit, other conflicts fixed in last commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,26 +33,26 @@ For Alluxio versions below 1.5.0, use the `alluxio-core-client` artifact.
 <dependency>
   <groupId>org.alluxio</groupId>
   <artifactId>alluxio-core-client-fs</artifactId>
-  <version>1.5.0</version>
+  <version>1.4.0</version>
```
Comment (aaudiber) [inline README.md]: @calvinjia This should always be less than 1.5.0, right?

Comment (calvinjia) [reply README.md]: Correct

File: README.md [hunk 2]
```diff
@@ -33,26 +33,26 @@ For Alluxio versions below 1.5.0, use the `alluxio-core-client` artifact.
...
-<dependency org="org.alluxio" name="alluxio" rev="1.6.0">
+<dependency org="org.alluxio" name="alluxio" rev="1.4.0">
   <artifact name="alluxio-core-client-fs" type="jar" />
 </dependency>
 ```
 
 #### SBT
 ```
-libraryDependencies += "org.alluxio" % "alluxio-core-client-fs" % "1.6.0"
+libraryDependencies += "org.alluxio" % "alluxio-core-client-fs" % "1.4.0"
```
Comment (calvinjia) [inline README.md]: Change these back to 1.6.0

File: README.md (hunk 1 above)
Comment (aaudiber) [reply README.md]: got it, done

**Pull request merged**